src/chemobjects/chemsys/System.h

```diff
@@ -222,7 +222,7 @@ class System{
   void TRANSLATE_FRAGMENT(double,VECTOR,int);
   void TRANSLATE_MOLECULE(double,VECTOR,int);
   void ROTATE_FRAGMENT(double, VECTOR,int);
-  void ROTATE_FRAGMENT(double, VECTOR,int, VECTOR);
+  void ROTATE_FRAGMENT(double, VECTOR,int, int);

```

Comment:
you have specialized the interface of the function. In principle, the "VECTOR center" could be any point, including the position of an atom. You version with "int" can be executed by passing the position of an atom around which the rotation occurs to the general function with VECTOR. Please use the function overloading to have both versions - with VECTOR and with int.